### ISSUE TITLE
Jest fails

### ISSUE THREAD

**Issue by bartonhammond** [bartonhammond/snowflake#81]
On BitRise, with .babelrc 
```
 extends: react-native/packager/react-packager/.babelrc
```
git this error
```
Runtime Error
Error: [BABEL] /bitrise/src/node_modules/react-native/jestSupport/env.js: Couldn't resolve extends clause of react-native/packager/react-packager/.babelrc in /bitrise/src/.babelrc
```

**Comment by joeltrew** [comment]
React native moved their babelrc config into its own 'react-native' babel preset, you can fix this issue by using that same preset like so in your .babelrc 
`{
  "presets": [ "react-native" ],
  "plugins": []
}
`

However I've found since 0.22, I've had issues running my tests as react-native babel preset seems to enforce strict mode on all tests which removes the ability to mock fetch requests

**Comment by bartonhammond** [comment]
fixed w/ release [0.0.14](https://github.com/bartonhammond/snowflake/releases/tag/0.0.14)
